pubspec.yaml

```diff
@@ -5,15 +5,14 @@ description: >-
   Opinionated, automatic Dart source code formatter.
   Provides an API and a CLI tool.
 repository: https://github.com/dart-lang/dart_style
-
 environment:
-  sdk: '>=2.12.0-0 <3.0.0'
+  sdk: ">=2.12.0-0 <3.0.0"
```

Comment:
Not sure where these double quotes came from.... I can revert if you want?

Reply:
Drop the `-0` while you're here!

Reply:
🤷‍♂️ I would really like to avoid changing more than what this PR is really about?